README.md

```diff
@@ -103,6 +103,25 @@ $ ./install-dev.sh
 
 The `install-dev.sh` script will build and push an image of the Tekton Dashboard to the Docker registry which you are logged into. Any Docker registry will do, but in this case it will push to Dockerhub. It will also build the static web content using `npm` scripts.
 
+### Supported backend options
+
+The dashboard backend supports a number of options through command line arguments.
+
+These options are documented below:
+
+| Argument | Description | Type | Default value |
+|---|---|---|---|
+| `-csrf-secure-cookie` | Enable or disable Secure attribute on the CSRF cookie | `bool` | `true` |
```

Comment:
in https://github.com/tektoncd/dashboard/pull/1403/files#diff-f785e9e46b9afd2567df7a658338db17R148 you have `--csrf-secure-cookie` (2 dashes)

we should be consistent about which we use

Reply:
fixed